mindmeld/auto_annotator.py

```diff
@@ -467,10 +502,26 @@ def supported_entity_types(self):  # pylint: disable=W0236
         Returns:
             supported_entity_types (list): List of supported entity types.
         """
-        return SPACY_ANNOTATOR_SUPPORTED_ENTITIES
+        spacy_supported_entities = [e.lower() for e in self.nlp.get_pipe("ner").labels]
+        supported_entities = set()
+        for entity in spacy_supported_entities:
+            entity = "person" if entity == "per" else entity
+            if entity == "misc":
```

Comment:
I think more documentation here would be helpful on why we skip misc

Reply:
Added